locations/spiders/prime_pubs.py

```diff
@@ -0,0 +1,46 @@
+# -*- coding: utf-8 -*-
+import json
+import re
+
+import scrapy
+
+from locations.items import GeojsonPointItem
+from locations.hours import OpeningHours
+
+
+class PrimePubsSpider(scrapy.Spider):
+    name = "prime_pubs"
+    allowed_domains = ['google.com']
+    start_urls = [
+        'https://spreadsheets.google.com/feeds/list/1idWVGZkrRLTEkm6eB0baD9J7G6u-4rLIud4BL52Md3Q/1/public/values?alt=json',
+    ]
+
+    def parse(self, response):
+        places = json.loads(response.body_as_unicode())
+
+        for place in places["feed"]["entry"]:
+            brand = place["gsx$storename"]["$t"]
+            if "FIONN MACCOOL'S" in brand:
+                brand = "FIONN MACCOOL'S"
```

Comment:
Can you fix the capitalization on these brand names please?

Reply:
Yes, I changed the brand names to camel case